nginx_config_reloader/__init__.py

```diff
@@ -78,8 +79,11 @@ def process_IN_MOVE_SELF(self, event):
         raise ListenTargetTerminated
 
     def handle_event(self, event):
+        if self.notifier.check_events(timeout=100):
```

Comment:
default notifier can be empty (`None`), so this line *could* error with attribute exception. :wink'

Reply:
https://github.com/ByteInternet/nginx_config_reloader/pull/43 take into account None notifier in handle_event #43